harley/column_functions.py

```diff
@@ -0,0 +1,7 @@
+from polars import Expr
+from typing import Any, List
+import polars as pl
+
+def multi_equals(cols: List[str], val: Any) -> Expr:
+    query = [pl.col(name) == val for name in cols]
```

Comment:
Do we want to warn / handle if `val = None`?

Reply:
what would the warning be, that some kind of `is_null` equivalent would be better?

Reply:
Yeah, but to be fair polars does it so maybe just leave it as is